formatter/src/formatter/node.rs

```diff
@@ -51,7 +51,34 @@ impl Formatter<'_> {
     }
 
     pub fn node_name(&mut self, name: &NodeName) {
-        self.printer.word(name.to_string());
+        match name {
+            NodeName::Path(_) | NodeName::Punctuated(_) => self.printer.word(name.to_string()),
+            NodeName::Block(block) => {
+                // Special case for props spreading with an empty element name
+                if let [syn::Stmt::Expr(syn::Expr::Range(range), _)] = block.stmts.as_slice() {
+                    match &range.end {
+                        // Spreading with identifier `{..ident}`
+                        Some(end) => {
+                            if let syn::Expr::Path(path) = &**end {
+                                if let Some(segment) = path.path.segments.last() {
+                                    let formatted = format!("{{..{}}}", segment.ident);
+                                    self.printer.word(formatted);
+                                    return;
+                                }
+                            }
+                        }
+                        // Spreading without identifier `{..}`
+                        None => {
+                            self.printer.word("{..}");
+                            return;
+                        }
+                    }
+                }
+
+                // Default handling if it doesn't match any specific pattern
+                self.printer.word(name.to_string())
+            }
+        }
     }
```

Comment:
Unrelated to your change, but could you add these to `attribute.rs` for completeness?

```rust
    #[test]
    fn prop_spreading_unnamed() {
        let f = format_attribute! {  {..}  };
        assert_snapshot!(f, @r#"{..}"#)
    }

    #[test]
    fn prop_spreading_named() {
        let f = format_attribute! {  { ..some_props }  };
        assert_snapshot!(f, @r#"{..some_props}"#)
    }
```